google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpanTest.java

```diff
@@ -154,11 +168,30 @@ public static void stopServer() throws InterruptedException {
 
   @Before
   public void setUp() throws Exception {
+    SpannerOptions.resetActiveTracingFramework();
+    SpannerOptions.enableOpenCensusTraces();
```

Comment:
1. Put the enable... in a @BeforeClass method
2. Put the reset... in a @AfterClass method

This will ensure that the configuration of this test class does not leak into any other tests.